jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/SubscribeSnsDialog.kt

```diff
@@ -87,6 +110,32 @@ class SubscribeSnsDialog(
         }
     }
 
+    private fun topicSelected(): String = view.topicSelector.selected()?.topicArn() ?: ""
+
+    private fun needToEditPolicy(existingPolicy: String?): Boolean {
+        existingPolicy ?: return true
+
+        // This does not actually make API calls
```

Comment:
What do you mean by this does not actually make API calls? The call to `simulateCustomPolicy` is an API call to IAM isn't it?

Reply:
nope:
```
 Simulate how a set of IAM policies and optionally a resource-based policy works with a list of API operations and AWS resources to determine the policies' effective permissions. The policies are provided as strings.

The simulation does not perform the API operations; it only checks the authorization to determine if the simulated policies allow or deny the operations. 
```

Reply:
yeah but you're still calling out over the wire to AWS - it's still a network call. When they say does not perform API calls I'm assuming it means that they don't call other AWS APIs to run the simulation.

Reply:
Ah yeah, looking at the implementation that is the case. You can't be sure without looking at the implementation since everything is modeled the same way, even things that actually don't make any api calls